traffic_router/core/src/main/java/com/comcast/cdn/traffic_control/traffic_router/core/router/TrafficRouter.java

```diff
@@ -86,6 +87,8 @@
 	private final ConsistentHasher consistentHasher = new ConsistentHasher();
 	private SteeringRegistry steeringRegistry;
 
+	private final Map<String, Geolocation> defaultGeolocations = new HashMap<String, Geolocation>();
```

Comment:
nitpick, but variable might be more clear named defaultGeolocationsOverride. Skimming the code I got this confused with defaultLocation (which is what we call a geolocation from Maxmind w/o a city or postal code)

Reply:
I've renamed defaultGeolocations to defaultGeolocationsOverride